src/services/userService.js

```diff
@@ -0,0 +1,54 @@
+let Sequelize = require('sequelize');
+
+
+function UserService(model) {
+  this.model = model;
+}
+
+UserService.prototype.createUser = async function(user) {
+  let data = await this.model.create(user);
+  return data.dataValues;
+};
+
+UserService.prototype.getUser = async function(id) {
+  let data = await this.model.findOne({
```

Comment:
Consider going through the codebase and carefully changing the "let" to "const" where applicable.
This is a part of error-prevention methodologies, so it is now a MUST.

Reply:
resolved